deploy/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "About Deno Deploy"
+description: "Guide to Deno Deploy features, comparison with Deploy Classic, and getting started instructions for deployment."
+---
+
+<a href="https://console.deno.com" class="docs-cta deploy-cta">Go to the Deno
+Deploy dashboard</a>
+
+Deno Deploy comes with an easy to use dashboard at
+[console.deno.com](https://console.deno.com). In this dashboard, you can create
+new Deno Deploy organizations that contain Deno Deploy apps.
+
+Within a single organization, you cannot mix Deno Deploy apps with Deploy
+Classic projects. You can switch between different organizations using the
+organization picker in the top left of the dashboard.
+
+## What is Deno Deploy?
+
+Deno Deploy is a serverless platform for running JavaScript and TypeScript
+applications in the cloud (or self-hosted on your own infrastructure). It
+provides a management plane for deploying and running applications through
+integrations like GitHub deployment.
```

Comment:
Would be good to also mention that Deploy can also perform build and deploys via our own CI now, and not only have builds run elsewhere like in GitHub actions

Reply:
have updated the copy a little